akka-stream/src/main/scala/akka/stream/impl/ActorMaterializerImpl.scala

```diff
@@ -36,6 +36,14 @@ abstract class ExtendedActorMaterializer extends ActorMaterializer {
     subflowFuser:      GraphInterpreterShell ⇒ ActorRef,
     initialAttributes: Attributes): Mat
 
+  // TODO not sure if needed exposing, I think so, at least right now -- ktoso
+  private[akka] def materialize[Mat](
```

Comment:
There will be probably some refactornings here, as default phase and phases should be remappable by users (otherwise it will not be possible to build new plugins). Not sure what is the best way yet to expose things, so fine to have it private now.